content/03-reference/01-tools-and-interfaces/07-preview-features/001-sql-server/010-sql-server-connection-string.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: 'SQL Server connection string'
+metaTitle: 'SQL Server connection string (Preview)'
+metaDescription: 'The SQL Server connection string.'
+---
+
+<TopBlock>
+
+The Microsoft SQL Server connection string follows the [JDBC standard](https://docs.microsoft.com/en-us/sql/connect/jdbc/building-the-connection-url?view=sql-server-ver15)
+
+The following example uses integrated security (Windows):
```

Comment:
Is this clear to everyone what "integrated security" means? I would have had no idea.

Reply:
I'd expect so (SQL Server on Windows), but I will add a link: https://docs.microsoft.com/en-us/dotnet/framework/data/adonet/sql/authentication-in-sql-server#:~:text=SQL%20Server%20supports%20two%20authentication,is%20tightly%20integrated%20with%20Windows.

Reply:
Yep, it uses from the client and server side the internal Windows libraries to authenticate to the server. On non-windows platform it does nothing, but in the future, if we want, we can enable the Kerberos feature for Tiberius, and then `IntegratedSecurity` would on Linux and macOS use the libgssapi and Kerberos for authentication.

This is a huge can of worms, so for now we just disable the feature during compile time.

Reply:
I've added some links and clarified 'Windows only' where it wasn't clear.